### ISSUE TITLE
[docs] clockspeed incorrect?

### ISSUE THREAD

**Issue by holstvoogd** [butterstick-fpga/butterstick-hardware#15]
All the documentation & production info at groupgets says it has a 60mhz clock.
However the schematics and example projects say 30mhz.

After some testing counting leds, I'm pretty sure it is _actually_ 30mhz, correct?

**Comment by gregdavill** [comment]
Yes, it is a 30MHz oscillator, the internal PLL boosts this upto 60MHz for USB or 125MHz for Ethernet.
